src/tests/dbus-tests/test_mdraid.py

```diff
@@ -33,6 +33,15 @@ class RAIDLevel(storagedtestcase.StoragedTestCase):
     chunk_size = 0
 
     def setUp(self):
+        try:
+            with open("/etc/redhat-release", "r") as f:
```

Comment:
Added a new commit to use ``platform.linux_distribution()`` in both places instead.